### ISSUE TITLE
The yolov5n mAP can't be reproducible as reported in README.MD.

### ISSUE THREAD

**Issue by wangziren1** [ultralytics/yolov5#6181]
### Search before asking

- [x] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Thank you for the great works.
I want to reproduce yolov5n mAP. At first, I used the command 
```
python3 train.py --data coco.yaml --cfg yolov5n.yaml --weights '' --batch-size 88 
```
I use as big batch size as possible to maximize the use of GPU. I get results as follows:
Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.276
Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.454
It is lower than 28.4 mAP@0.50:0.95 as reported in README. Then I search the issue and use the following command:
```
python3 train.py --data coco.yaml --cfg yolov5n.yaml --weights '' --batch-size 88 --hyp hyp.scratch-low.yaml --linear
```
I get results as follows:
Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.277
Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.454
Still, It is almost same as the first one and can't reach 28.4mAP.
Can you make some suggestions about that? Thank you.

**Comment by glenn-jocher** [comment]
@wangziren1 you should be able to reproduce with your second command. The official training command for this model is:

```bash
python train.py --data coco.yaml --batch 64 --weights '' --cfg yolov5n.yaml --epochs 300 --project yolov5_v6 --name yolov5n --img 640 --hyp hyp.scratch-low.yaml --linear --device 0
```

One difference is you may need to use the segmentation dataset rather than the box dataset, but I'm not sure. The COCO segmentation dataset is:
```
bash data/scripts/get_coco.sh
```
Except that L13 should say `f='coco2017labels-segments.zip'`

**Comment by glenn-jocher** [comment]
@wangziren1 can you try with this new dataset and report back if there's a difference?

**Comment by imyhxy** [comment]
@glenn-jocher Hi, I have trained YOLOv5n with segment labels twice, both get 28.0 mAP on COCO dataset. I also upload my [train.log](https://github.com/ultralytics/yolov5/files/8020020/train.log) here. Let me know if I missed anything or the 0.4 mAP is an acceptable training variance.

P.S. My `albumentations` is `1.1.0` which does not match the required version `1.0.3`. Not sure how much of an impact this will have.

**Comment by glenn-jocher** [comment]
@imyhxy the v6.0 model trainings are public here. You can see the training command to reproduce, it's using hyp-scratch-low with --linear. I can't tell from the logs whether Albumentations was used or not though. With the smaller models generally less augmentation produces better results as they don't overfit as early.
https://wandb.ai/glenn-jocher/YOLOv5_v60_official

Reproducibility is also not guaranteed, so you should expect to see minor variations in mAP, i.e. maybe a few tenths of a percent, i.e. +/- 0.002 1 sigma maybe, so a 0.004 deviation might be a 2 sigma event (i.e. only 5% probability of occuring).

<img width="572" alt="Screenshot 2022-02-08 at 13 30 38" src="https://user-images.githubusercontent.com/26833433/152987429-a1e41fda-d79d-40fb-afea-71011d892c90.png">

**Comment by imyhxy** [comment]
@glenn-jocher Thanks for your reply, the wandb log will be very useful. 🚀

**Comment by diangu001** [comment]
> @wangziren1你应该可以用你的第二个命令来重现。这个模型的官方训练命令是：
> 
> ```shell
> python train.py --data coco.yaml --batch 64 --weights ' ' --cfg yolov5n.yaml --epochs 300 --project yolov5_v6 --name yolov5n --img 640 --hyp hyp.scratch-low。 yaml --linear --device 0
> ```
> Why not load weights
> 一个区别是您可能需要使用分段数据集而不是框数据集，但我不确定。COCO 分割数据集是：
> 
> ```
> bash data/scripts/get_coco.sh
> ```
> 
> 除了 L13 应该说`f='coco2017labels-segments.zip'`

**Comment by wangtao0219** [comment]
@imyhxy Did you reproduce successfully?
python train.py --data coco.yaml --batch 64 --weights '' --cfg yolov5n.yaml --epochs 300 --project yolov5_v6 --name yolov5n --img 640 --hyp hyp.scratch-low.yaml --linear --device 0 ，and f='coco2017labels-segments.zip'.

I can't reach 28.4mAP. Always a difference of 0.4%（28.0%）

**Comment by glenn-jocher** [comment]
@wangtao0219 https://wandb.ai/glenn-jocher/YOLOv5_v61_official YOLOv5n mAP is 28.0
